src/components/FlowPlayer/FlowPlayer.scss

```diff
@@ -32,7 +31,7 @@ $c-video-player-progress-inner-bg: $color-teal-bright;
 		padding: 0.5rem 1rem;
 
 		background-color: $color-teal-bright;
-		box-shadow: 0 0 5px 0px rgba(#222, 0.5);
+		box-shadow: 0 0 5px 0 rgba(#222, 0.5);
```

Comment:
Best deze `box-shadow` als variabele opslaan in `styles/settings/variables` en de `#222` in `styles/settings/colors` aangezien het meerdere keren hergebruikt wordt.